pagopa/pn-frontend
# fix(PN-9089): [FE] QRcode reading error message mitigation

## Short description
Updated AccessDenied component to show message when QRcode is read by the wrong receiver or without proxies.

## List of changes proposed in this pull request
- Added icon props to make the AccessDenied component behave differently in this case
- Passed icon prop as <IllusQuestion /> to AARGuard in PF and PG
- Created IllusQuestion to import it in AARGuard component
- Moved setting item method from AARGuard in RouteGuard in PG

- Updated README.md with new command for running single file test

## How to test
Open the notification link via qrcode of an aar (both for PG and for PF); check if the error message (icon, title, subtitle and button "vai alla homepage") is shown.
